app/backend/src/couchers/resources.py

```diff
@@ -38,12 +41,56 @@ def get_language_dict():
     """
     Get list of allowed languages as a dictionary of {code: name}.
     """
-    with open(resources_folder / "languages.json", "r") as f:
-        return {language["code"]: language["name"] for language in json.load(f)}
+    with session_scope() as session:
+        return {language.code: language.name for language in session.query(Language).all()}
 
 
 def language_is_allowed(code):
     """
     Check a language code is valid
     """
     return code in get_language_dict()
+
+
+def copy_resources_to_database(session):
+    """
+    Syncs the source-of-truth data from files into the database. Call this at the end of a migration.
+
+    Foreign key constraints that refer to resource tables need to be set to DEFERRABLE.
+
+    We sync as follows:
+
+    1. Lock the table to be updated fully
+    2. Defer all constraints
+    3. Truncate the table
+    4. Re-insert everything
+
+    Truncating and recreating guarantees the data is fully in sync.
+    """
+    with open(resources_folder / "regions.json", "r") as f:
+        regions = [(region["alpha3"], region["name"]) for region in json.load(f)]
+
+    with open(resources_folder / "languages.json", "r") as f:
+        languages = [(language["code"], language["name"]) for language in json.load(f)]
+
+    timezone_areas_file = resources_folder / "timezone_areas.sql"
+
+    if not timezone_areas_file.exists():
+        timezone_areas_file = resources_folder / "timezone_areas.sql-fake"
+        logger.info(f"Using fake timezone areas")
```

Comment:
How should this work?

Through an env var? Like this but raise an exception if in prod?

Reply:
Hard error if timezone_areas.sql doesn't exists, with a pointer to the README that describes how to download it.